Lib/cu2qu/__init__.py

```diff
@@ -209,14 +207,15 @@ def curve_to_quadratic(curve, max_err):
     """
 
     curve = [complex(*p) for p in curve]
-    spline = None
+
     for n in range(1, MAX_N + 1):
         spline = cubic_approx_spline(curve, n, max_err)
         if spline is not None:
             break
```

Comment:
Same comment from curves_to_quadratic applies here, now that I think about it. I think it would be clearer to return here instead of breaking and do away with the else (always raise an error if nothing was returned).